server/pulp/server/webservices/http.py

```diff
@@ -218,10 +162,7 @@ def uri_path():
     Return the current URI path
     @return: full current URI path
     """
-    if _is_django():
-        return unicode(_get_wsgi_environ()['REQUEST_URI'])
-    else:
-        return web.http.url(web.ctx.path)
+    return unicode(_thread_local.wsgi_environ['REQUEST_URI'])
```

Comment:
This won't work the way it is written because the line that set wsgi_environ in thread locals is now gone. It was here: https://github.com/pulp/pulp/pull/1830/files#diff-9a48cb728a264356eaf4f26c0f2286e5L68

Look at the code that is calling this and determine if everything can be gotten from the django request object.